static/app/views/issueList/overview.tsx

```diff
@@ -1063,19 +1126,24 @@ class IssueListOverview extends React.Component<Props, State> {
   };
 
   onActionTaken = (itemIds: string[]) => {
+    const actionTakenGroupData = itemIds.map(id => GroupStore.get(id));
```

Comment:
```suggestion
    const actionTakenGroupData = itemIds.map(id => GroupStore.get(id) as Group);
```
so we don't cast on the next line